apps/heureka/src/components/services/ComponentInstancesList.jsx

```diff
@@ -0,0 +1,123 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useState, useMemo } from "react"
+import { useQuery } from "@tanstack/react-query"
+import {
+  ContentHeading,
+  Pagination,
+  Container,
+  Stack,
+  DataGrid,
+  DataGridRow,
+  DataGridCell,
+  DataGridHeadCell,
+} from "@cloudoperators/juno-ui-components"
+import { useGlobalsQueryClientFnReady, useGlobalsQueryOptions, useGlobalsActions } from "../StoreProvider"
+import LoadElement from "../shared/LoadElement"
+import { severityString, highestSeverity } from "../shared/Helper"
+
+const ComponentInstancesList = ({ serviceName }) => {
+  const queryOptions = useGlobalsQueryOptions("ComponentInstancesOfService")
+  const { setQueryOptions } = useGlobalsActions()
+  const queryClientFnReady = useGlobalsQueryClientFnReady()
+  const [currentPage, setCurrentPage] = useState(1)
+
+  const { data, isLoading } = useQuery({
+    queryKey: [
+      "ComponentInstancesOfService",
+      {
+        ...queryOptions,
+        filter: { serviceName: [serviceName] },
+      },
+    ],
+    enabled: !!queryClientFnReady && !!serviceName,
+  })
+
+  const items = useMemo(() => {
```

Comment:
I think you probably do not need use memo here do you?

Reply:
I think the internal handling of useMemo consumes more performance resources than the benefits it provides in this case.

Reply:
I remove it, since I think the computation here is not complicate.